packages/myst-frontmatter/src/frontmatter/validators.ts

```diff
@@ -32,19 +32,23 @@ import type {
   SiteFrontmatter,
   TextRepresentation,
   Venue,
-  SiteDesign,
 } from './types';
 
-export const SITE_FRONTMATTER_KEYS = ['title', 'description', 'venue', 'keywords', 'design'];
-export const PROJECT_FRONTMATTER_KEYS = [
+export const SITE_FRONTMATTER_KEYS = [
+  'title',
+  'description',
   'authors',
+  'venue',
+  'github',
```

Comment:
Are we removing these from the site? Or maybe I am not sure about what `SITE_FRONTMATTER_KEYS` is?

Reply:
`SITE_FRONTMATTER_KEYS` is just a list of the keys defined on the `SiteFrontmatter` type. The only change here is _adding_ `authors` and `github` to site frontmatter.

Reply:
Corresponding type change here: https://github.com/executablebooks/mystjs/pull/99/files#diff-0efda934c1ea585a450b89b6d34781b6e78c12ada734af7d546f40970eb1730eR96-R102

Reply:
Ok, that is much clearer. I left a note about maybe changing the type structure (not sure if the validators would also be clearer if they were a bit more repeated rather than inherited now that site --> project isn't a thing?)